Packages/Netherlands3D/T3DPipeline/Runtime/Scripts/ParseCityJSON/CityJSONStructure/CityObject.cs

```diff
@@ -63,14 +63,32 @@ public class CityObject : MonoBehaviour
 
         public UnityEvent CityObjectParsed { get; private set; } = new UnityEvent();
 
+        private bool includeInExport;
+        public bool IncludeInExport
+        {
+            get
+            {
+                return includeInExport;
+            }
+            set
+            {
+                if (value)
```

Comment:
is dit een null check?

Reply:
nee, dit is zodat je met 
IncludeInExport  = false; 
de omliggende gebouwen uit de exporter kan halen